site/public/js/sql-toolbox.js

```diff
@@ -52,8 +55,13 @@ async function runSqlQuery() {
         header.appendChild(header_row);
         table.appendChild(header);
         const body = document.createElement('tbody');
+        let rownum = 1;
         data.forEach((row) => {
```

Comment:
You can amend this to be:

```suggestion
        data.forEach((row, idx) => {
```

and then on line 62 you could do `cell.textContent = idx + 1;` and not have and track the `rownum` variable.